### PR TITLE
Fix tests after #11239

### PR DESCRIPTION
**Pull request by ksercs**
before:
image
after:
image

### CODE REVIEW

**Review by DokaRus — DISMISSED** [review]

**Review by alexander-kotov-dx — DISMISSED** [review]

**Review by San4es — DISMISSED** [review]

**Review by DokaRus — DISMISSED** [review]

File: styles/widgets/material/toolbar.material.less [hunk 1]
```diff
@@ -192,7 +192,6 @@
     .dx-button-has-text:not(.dx-button-back) {
         .dx-icon {
             .dx-icon-sizing(@MATERIAL_BUTTON_ICON_SIZE);
-            .dx-icon-margin(6px);
```
Comment (alexander-kotov-dx) [inline styles/widgets/material/toolbar.material.less]: Is it changed the appearance of other button types in dxToolbar?

Comment (ksercs) [reply styles/widgets/material/toolbar.material.less]: fixed it

**Review by alexander-kotov-dx — DISMISSED** [review]

File: styles/widgets/generic/dropDownButton.generic.less [hunk 2]
```diff
@@ -2,8 +2,18 @@
 @import (once) "./popup.generic.less";
 @import (once) "./list.generic.less";
 
+.dx-button {
```
Comment (San4es) [inline styles/widgets/generic/dropDownButton.generic.less]: This style will be applied for all buttons, will not?

Comment (ksercs) [reply styles/widgets/generic/dropDownButton.generic.less]: fixed it

**Review by San4es — APPROVED** [review]

**Review by alexander-kotov-dx — APPROVED** [review]

**Pull request merged**